huxley/www/js/components/AdvisorProfileView.js

```diff
@@ -364,8 +368,13 @@ var AdvisorProfileView = React.createClass({
   _handleSuccess: function(data, status, jqXHR) {
     this.setState({
       errors: {},
-      loading: false
+      loading: false,
+      success: true
     });
+
+    setTimeout(() => {
```

Comment:
We should store this timeout and cancel it if the component unmounts, and also if the user clicks submit again.